recipe/meta.yaml

```diff
@@ -26,9 +27,9 @@ requirements:
   run:
     - python >=3.6
     - cryptography >=3.2
```

Comment:
`cryptography` isn't required. @chenghlee Do we need it here?

Reply:
I don't think it should be necessary - removed it.